amethyst_renderer/src/pipe/effect/mod.rs

```diff
@@ -248,6 +257,15 @@ impl<'a> EffectBuilder<'a> {
                 Stencil::default(),
             ));
         }
+        // OSX doesn't seem to work without a depth test, so here's a workaround.
```

Comment:
Might be better for future refactoring to just use a single method for these two